Sources/SwiftDoc/API.swift

```diff
@@ -55,3 +55,9 @@ extension Subscript: API {
 
 extension Typealias: API {}
 extension Variable: API {}
+
+public enum AccessLevel: String, Codable {
```

Comment:
One fancy thought: Instead of using `String` as the raw representable, we could also use `Int` and make the enum conform to `Comparable`. Then we could do nice expressions like `if minimumAccessLevel >= accessLevel`.

Reply:
BTW, if it goes that route and supports `Comparable`, should have a note added that it won't be needed with Swift 5.3 thanks to [SE-0266](https://github.com/apple/swift-evolution/blob/master/proposals/0266-synthesized-comparable-for-enumerations.md).

Reply:
I've already started work with the latest 5.3 snapshot to get [package resources](https://github.com/apple/swift-evolution/blob/master/proposals/0271-package-manager-resources.md) working, so I'll take a look at using automatic `Comparable` synthesis with an explicit implementation for `<` in a conditional compilation block.